tests/django/test_views.py

```diff
@@ -138,3 +139,55 @@ def test_returns_errors_and_data():
 
     assert data["data"]["hello"] == "strawberry"
     assert data["data"]["alwaysFail"] is None
+
+
+class ListHandler(logging.Handler):
+    def __init__(self, *args, **kwargs):
+        super(ListHandler, self).__init__(*args, **kwargs)
+
+        for level in logging._nameToLevel.keys():
+            setattr(self, level.lower(), [])
+
+    def emit(self, record):
+        msg = record.getMessage()
```

Comment:
Maybe we can mock `log_errors` or `self.logger.error` in the tests instead of creating a custom logger like this?